iosApp/iosApp/Pages/More/MoreToggle.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  MoreToggle.swift
+//  iosApp
+//
+//  Created by Melody Horn on 4/28/25.
+//  Copyright © 2025 MBTA. All rights reserved.
+//
+
+import Shared
+import SwiftUI
+
+struct MoreToggle: View {
+    let label: String
+    let setting: Settings
+    @ObservedObject var settingsCache: SettingsCache
```

Comment:
You can’t combine `@ObservedObject` and `@Environment` (they can’t each wrap the property independently, one of them has to wrap the other wrapper, and neither of them wants to do that), and it’s critical that we rerender the view when the cache content changes, so this has to be passed explicitly, which is unfortunate.